airbyte-cdk/java/airbyte-cdk/db-sources/src/main/kotlin/io/airbyte/cdk/integrations/source/relationaldb/state/SourceStateIterator.kt

```diff
@@ -45,8 +47,11 @@ open class SourceStateIterator<T>(
                 stateMessage!!.withSourceStats(
                     AirbyteStateStats().withRecordCount(recordCount.toDouble())
                 )
+                LOGGER.info { "sending state message, with count per stream: $streamRecordCount " }
```

Comment:
How noisy is this going to be? How many state messages do we send, on average, per sync?

Reply:
We are sending 1 state message per 10000 records - As of current behavior platform would log once every 5000 records to log how many records have been processed, so I think it's not going to be too bad, given the log itself should be concise (versus printing out the entire state message).